package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Kindly add `dev` configuration. This will ensure that all configurations are in place.  You can do this by adding ` "dev": "webpack --mode development",` in your script file ⬇️ 

![scripts_requirement (1)](https://user-images.githubusercontent.com/32361963/111418734-45be2500-86f9-11eb-9734-93b4997f1c78.png)